packages/ripple-binary-codec/test/fixtures/definitions-with-diff-payment.json

```diff
@@ -0,0 +1,2337 @@
+{
```

Comment:
A thought - it's really hard to parse these definitions.json files and figure out what the diff is with the existing definitions.json. Could we instead just use the existing definitions.json, and modify it in the JS code, instead of having these massive fixtures that are hard to understand?

Reply:
That seems like a good suggestion :) - Done.